Evaluator/RealTime/instant_fluctuations_evaluator.py

```diff
@@ -2,6 +2,7 @@
 OctoBot Tentacle
 
 $tentacle_description: {
+    "package_name": "OctoBot-Tentacles",
```

Comment:
Need to remove this line: package-name is auto generated by tentacle manager when installing (it's used to know the origin of a tentacle)

Reply:
My bad.